featuretools/tests/primitive_tests/test_agg_feats.py

```diff
@@ -635,32 +635,15 @@ def test_stacking_multi(es):
 
 
 def test_use_previous_pd_dateoffset(es):
-    total_events_last_20_years = ft.Feature(es["log"]["id"],
-                                            parent_entity=es["customers"],
-                                            use_previous="20 years",
-                                            primitive=Count)
-
-    feature_matrix = ft.calculate_feature_matrix([total_events_last_20_years], es)
-
-    total_events_last_20_years_pd = ft.Feature(es["log"]["id"],
-                                               parent_entity=es["customers"],
-                                               use_previous=pd.DateOffset(years=20),
-                                               primitive=Count)
-
-    feature_matrix_2 = ft.calculate_feature_matrix([total_events_last_20_years_pd], es)
-
-    assert (feature_matrix["COUNT(log, Last 20 Years)"] == [10, 5, 2]).all()
-    assert (feature_matrix["COUNT(log, Last 20 Years)"] == feature_matrix_2["COUNT(log, Last 20 Years)"]).all()
-
-    since_str = (pd.to_datetime('today') - pd.tseries.offsets.BDay(100))
-    total_events_last_100_bday = ft.Feature(es["log"]["id"],
-                                            parent_entity=es["customers"],
-                                            use_previous=pd.tseries.offsets.BDay(100),
-                                            primitive=Count)
-    since_str = since_str.strftime('%Y-%m-%d %H:%M:%S')
-    feature_matrix_3 = ft.calculate_feature_matrix([total_events_last_100_bday], es)
+    time_diff = (pd.Timestamp('today') - pd.Timestamp('2011-04-09 10:32:00')).total_seconds()
```

Comment:
Might be better to set a constant time_diff instead of anchoring on the current date

Reply:
It's currently anchored on this date as use_previous depends on the current date to run. If it was constant, then the result of the test could change to all zeros rather than the nonzero answer this test is trying to look for.

Reply:
If we changed `cutoff_time` to be a constant time as well instead of the current time than we could avoid anchoring on the current date.